.rubocop.yml

```diff
@@ -81,6 +81,16 @@ Style/DotPosition:
 Style/DoubleNegation:
   Enabled: false
 
+Style/SymbolArray:
```

Comment:
`SymbolArray` cop is enabled in default.
This cop changes all `[]` into `%i`.
It has big effect on entire project.
So I disabled.

ref:https://github.com/bbatsov/rubocop/issues/4124